starwars/src/Components/Cards.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import StarCharacters from "./StarCharacters";
+import PropTypes from 'prop-types';
+
+const Cards = props => {
```

Comment:
Would be a good spot to use object destructuring, since `props` is just an object afterall.

```
const Data = ({ characters }) =>  {
```

Then on line 9, reference `characters.map()`